docs/docs/command-line-interface.mdx

```diff
@@ -23,7 +23,7 @@ The following commands are relevant to all assistants built with Rasa.
 | `rasa run actions`      | Starts an action server using the Rasa SDK.                                                        |
 | `rasa test e2e`         | Runs end-to-end testing fully integrated with the action server that serves as acceptance testing. |
 | `rasa data split nlu`   | Performs a 80/20 split of your NLU training data.                                                  |
-| `rasa data validate`    | Checks the domain, NLU and conversation data for inconsistencies.                                  |
+| `rasa data validate`    | Checks the domain, NLU, business logic and conversation data for inconsistencies.                                  |
```

Comment:
wouldn't it be more clear if we said "flows" instead of business logic?

Reply:
Sure, I can change that.